internal/gomod/stdlib.go

```diff
@@ -0,0 +1,232 @@
+package gomod
+
+// IsStandardlibPackge determines whether a package is in the standard library
+// or not. At this point, it checks whether the package name is one of those
+// that is found from running "go list std" in the latest released go version.
+func IsStandardlibPackge(pkg string) bool {
+	_, ok := standardLibraryMap[pkg]
+	return ok
+}
+
+var contained interface{}
```

Comment:
`var contained struct{}{}` is a bit more idiomatic (and you can inline it since it's a zero-width type).